neon-breaker/js/powerups.js

```
@@ -107,34 +107,28 @@ class PowerUpManager {
-        (gameState.scoreMultiplier || 1) / 2,
-      );
-      break;
-    case 'shield':
-      gameState.hasShield = false;
-      break;
+      case 'bigPaddle':
+        gameState.paddle.width = powerUp.originalWidth;
+        break;
+      case 'slowBall':
+        // Restore original ball speed
+        const currentSpeed = Math.sqrt(gameState.ball.dx * gameState.ball.dx + gameState.ball.dy * gameState.ball.dy);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)